packages/manager/modules/sms/package.json

```diff
@@ -30,7 +30,9 @@
     "moment": "^2.22.2"
   },
   "devDependencies": {
-    "@ovh-ux/component-rollup-config": "5.0.0-beta.5"
+    "@ovh-ux/component-rollup-config": "5.0.0-beta.5",
+    "bootstrap": "~3.3.7",
+    "ovh-ui-kit": "^2.23.1"
```

Comment:
Thanks for adding the same version from the `ovh-manager-telecom`.

Just for your information:
```sh
$ npm info ovh-ui-kit | grep latest
latest: 2.25.0
```

Reply:
Not in the develop branch version, no ?